### ISSUE TITLE
Issue with yarn, compiled with npm but getting installation error after upload

### ISSUE THREAD

**Issue by AndreaPro** [jellyfin/jellyfin-tizen#88]
Hello, I've done the following: 
- deployed Ubuntu 20.04.2 LTS inside VirtualBox
- installed Tizen Studio with IDE from .bin file downloaded from samsung website
- installed git, nodejs and yarn with `sudo apt install git nodejs yarn`
- followed all the steps until the **Prepare Interface** section, here I get this error when running `JELLYFIN_WEB_DIR=../jellyfin-web/dist yarn install`:
`00h00m00s 0/0: : ERROR: [Errno 2] No such file or directory: 'install'`
I've compiled using `JELLYFIN_WEB_DIR=../jellyfin-web/dist npm install` instead. I don't know if this is the source of the next issue...
- I've obtained the .wgt file, I have established connection with TV and enabled "Permit to install applications" successfully
- when I run `tizen install -n Jellyfin.wgt -t UE50NU7090` I get this error:
```
Transferring the package...
Transferred the package: /home/test/jellyfin-tizen/Jellyfin.wgt -> /home/owner/share/tmp/sdk_tools/tmp
Installing the package...
--------------------
Platform log view
--------------------
install wgt
package_path /home/owner/share/tmp/sdk_tools/tmp/Jellyfin.wgt
was_install_app return WAS_TRUE
app_id[wgt] install start
app_id[wgt] download failed[116]
spend time for wascmd is [664]ms
cmd_ret:0
Failed to install Tizen application.
Total time: 00:00:08.347
```

Any help would be appreciated

**Comment by dmitrylyzo** [comment]
https://developer.youi.tv/5.7/Content/Tizen_DevTopics/H1BuildingInstallingAppWidgets.htm

116 | Not enough free space on the target device.
-- | --

If you really don't have much free space, try removing the fonts (`*.woff*`) from `www` and build WGT.

**Comment by AndreaPro** [comment]
I have managed to free up some space and now it is working!
I should expect some issues having used npm instead of yarn? Any ideas why this error comes up?

**Comment by dmitrylyzo** [comment]
You can use `npm` instead of `yarn` without any side effects. _This is currently possible because I don't use lock-files._

I can't tell why that error occurred. I am using Yarn 1.22.5 and I don't have that error.

What output of `yarn -v`?

**Comment by AndreaPro** [comment]
> You can use `npm` instead of `yarn` without any side effects. _This is currently possible because I don't use lock-files._
> 
> I can't tell why that error occurred. I am using Yarn 1.22.5 and I don't have that error.
> 
> What output of `yarn -v`?

If I type `yarn-v` I get: `ERROR: There are no scenarios; must have at least one.`

If I type `yarn --version` I get: `0.32+git`

**Comment by dmitrylyzo** [comment]
> If I type yarn --version I get: 0.32+git

Looks like you installed `cmdtest`. If you don't need `cmdtest`, remove it.

I installed Yarn following [this](https://linuxize.com/post/how-to-install-yarn-on-ubuntu-20-04/#installing-yarn-on-ubuntu).
There is also `yarnpkg` in Ubuntu repo, but you will probably have to run it as `yarnpkg install` (_not tested_).

At the same time, `npm` is sufficient, and we will probably move to it (someday), as in other projects.

**Comment by AndreaPro** [comment]
Yes probably the issue was that I have installed Yarn without adding the repository. Maybe could be a good idea to add a link to the yarn installation to help other users to avoid the same issue

**Comment by cipitaua** [comment]
yes on ubuntu you have to replace `yarn` with `yarnpkg`
